open-sphere-base/core/src/main/java/io/opensphere/core/control/ui/ToolbarManager.java

```diff
@@ -595,4 +595,17 @@ public void setVisible(IconButton button, boolean vis)
         /** South. */
         SOUTH;
     }
+
+    /**
+     * Listener for showIconButtonText preference updates.
+     */
+    private final PreferenceChangeListener myTextListener = new PreferenceChangeListener()

```

Comment:
Please put this at the top section of the class with the other member variables.